stable/contour/values.yaml

```diff
@@ -0,0 +1,63 @@
+deployment:
+  replicas: 2
+contour:
+  image:
+    repository: gcr.io/heptio-images/contour
+    tag: v0.5.0
```

Comment:
Can you please update to a newer version of contour. I see v0.9.0 has been released.

Reply:
@mattfarina Upgraded contour and envoy version to latest.